printable-recipe-generator/src/main.ts

```diff
@@ -3,8 +3,63 @@ import * as process from 'node:process';
 import path from 'path';
 import type { Data } from 'ejs';
 import { render as renderTemplate } from 'ejs';
+import fetch from 'node-fetch';
+
+//load Contributors
+async function getChefs(): Promise<unknown> {
```

Comment:
You shouldn't return a `Promise<unknown>` here really.

The chefs file has a defined format, so you should define each entry as an `interface` and then return a `Record<string, ChefData>` type.

Ideally you'd define it in Zod and use `zod.parse` to validate the data at runtime, but technically it's enough to simply cast the `unknown` value you get back from json (untested code, but it's something like this):

```typescript
  return (await resp.json()) as Record<string, ChefData>
```

Reply:
Thanks @fredex42. Yeah thats what I was checking with you that whether it is fine to not to have any interface/type declared and supressing them with ts-ignore tags but it seems we should use interface to avoid those things 👍